src/pytorch_lightning/callbacks/callback.py

```diff
@@ -82,7 +82,12 @@ def on_train_batch_start(
     def on_train_batch_end(
         self, trainer: "pl.Trainer", pl_module: "pl.LightningModule", outputs: STEP_OUTPUT, batch: Any, batch_idx: int
     ) -> None:
-        """Called when the train batch ends."""
+        """Called when the train batch ends.
+
+        Note:
+            The value ``outputs["loss"]`` here will be the normalized value w.r.t ``accumulate_grad_batches`` of the
+            loss returned from ``training_step``.
```

Comment:
@rbracco I pushed your note to the docstring here directly. It will show up automatically in the docs where you originally put it.